_includes/benchmarks.html

```diff
@@ -1,5 +1,9 @@
 <!-- generated by JuliaLang/Microbenchmarks/bin/table.jl -->
 
+<head>
+  <meta name="description" content="These are benchmarks of Julia as compared to other popular programming langauges in terms of popular mathematical algorithms. Learn more here."/>
```

Comment:
"benchmarks of Julia as compared to" should be "benchmarks of Julia compared to"